build.sbt

```diff
@@ -44,44 +43,32 @@ lazy val coverageSettings: Seq[Setting[_]] = {
   )
 }
 
-val compile = Seq(
-  ws,
-  "uk.gov.hmrc"       %% "bootstrap-backend-play-28" % bootstrapPlayVersion,
-  "com.typesafe.play" %% "play-json-joda"            % "2.10.0-RC7"
-)
-
-def test(scope: String = "test,it"): Seq[ModuleID] = Seq(
-  "uk.gov.hmrc"            %% "bootstrap-test-play-28" % bootstrapPlayVersion % scope,
-  "org.jsoup"              %  "jsoup"                  % "1.15.3"             % scope,
-  "org.scalatestplus"      %% "mockito-3-3"            % "3.2.2.0"            % scope
-)
-
 def oneForkedJvmPerTest(tests: Seq[TestDefinition]): Seq[Group] = tests map {
   test => Group(test.name, Seq(test), SubProcess(
     ForkOptions().withRunJVMOptions(Vector("-Dtest.name=" + test.name))))
 }
 
+ThisBuild / scalaVersion := "2.13.12"
```

Comment:
scala version can go up to 2.13.14

Reply:
@derickdamoah I tend to align to latest versions shown within tax catalogue even if not latest version.